pkg/base/linuxutils/lsblk/lsblk.go

```diff
@@ -161,8 +161,13 @@ func (l *LSBLK) GetBlockDevices(device string) ([]BlockDevice, error) {
 // Receives an instance of drivecrd.Drive struct
 // Returns drive's path based on provided drivecrd.Drive or error if something went wrong
 func (l *LSBLK) SearchDrivePath(drive *api.Drive) (string, error) {
+	// device path might be already set by hwmgr
```

Comment:
adding this code back seems violate @prasant123 previous fix code intention.   if device is in drive CR,  then the device name in driveCR is returned here,  this might be a wrong name due to device name change.

Reply:
ok i'll remove the test case and put back his orig code.
thanks

Reply:
@libzhang  - do you have any idea why all the PR checks are no longer on this PR?

Reply:
I am not sure,  maybe there is some bug in the our release branch workflow.